docs/components/modeler/bpmn/ad-hoc/ad-hoc.md

```diff
@@ -0,0 +1,75 @@
+---
+id: ad-hoc
+title: "Ad-hoc"
+description: "The ad-hoc marker is only available for embedded subprocesses, allowing more flexibility while executing inner elements."
+---
+
+The ad-hoc marker (represented by a **~** tilde character) is only available
+for [embedded subprocesses](../embedded-subprocesses/embedded-subprocesses.md), and are therefore called **ad-hoc subprocesses**.
+Compared to regular subprocesses, ad-hoc subprocesses allow more flexibility for executing inner elements.
+
+![A demo process with an ad-hoc subprocess. Some elements inside the subprocess are active.](assets/ad-hoc-subprocess.png)
+
+The inner elements of an ad-hoc subprocess are not connected to a start or end event. Each element can be executed
```

Comment:
We don't use "inner element" anywhere else in the docs -- can we define that here, even if it is brief? For example, "The inner elements (elements that are not start or end events) of an ad-hoc subprocess..." 

Or, do we just mean the tasks/gateways?

Reply:
inner elements = elements inside the ad-hoc subprocess

If the term is not clear, you could replace or remove it. I would leave this up to you. 

We use the same term in the [multi-instance docs](https://preview.docs.camunda.cloud/pr-4729/docs/next/components/modeler/bpmn/multi-instance/).

Reply:
I will open a backlog issue to revisit these references 👍